crates/autopilot/src/decoded_settlement.rs

```diff
@@ -254,6 +259,115 @@ impl DecodedSettlement {
             }
         })
     }
+
+    /// Returns the list of partial limit orders with their fees
+    pub fn order_executions(
```

Comment:
naming nit: this only returns partially fillable order executions, I think the name should reflect that.

Reply:
I see, I think this returns the list of order executions to update (since the surplus fee isn't known until there is a transaction on-chain).

Reply:
@cowfee - ping on naming nit.

Reply:
https://github.com/cowprotocol/services/pull/1425/commits/95c81c85b5cbd401aa8b983c7844004a67098777